packages/react-components/src/components/Picker/Trigger.tsx

```diff
@@ -55,9 +57,15 @@ export const Trigger: React.FC<ITriggerProps> = ({
       }
     };
 
-    document.addEventListener('keydown', onKeyDown);
+    if (!disabledSearch) {
+      document.addEventListener('keydown', onKeyDown);
+    }
 
-    return () => document.removeEventListener('keydown', onKeyDown);
+    return () => {
+      if (!disabledSearch) {
+        document.removeEventListener('keydown', onKeyDown);
+      }
+    };
   }, []);
```

Comment:
I think 'disabledSearch' should be added to the dependency table. Linter didn't complain about that?